src/routes/StandaloneMain.tsx

```diff
@@ -62,6 +63,21 @@ export function StandaloneMain() {
         isManagedSidebar
         defaultManagedSidebarIsOpen={true}
       >
+        <Banner variant="info">
+          <Flex
+            spaceItems={{default: 'spaceItemsSm'}}
+            justifyContent={{default: 'justifyContentCenter'}}
+          >
+            <FlexItem>
+              <InfoCircleIcon />
+            </FlexItem>
+            <FlexItem>
+              Please use{' '}
+              <a href="https://forms.gle/M2CtyneF3iaMT5UVA">this form</a> to
```

Comment:
can you add `target="_blank"` in the `<a>` so that the survey will open in a new tab

Reply:
oh yeah good call

Reply:
added